Source/santactl/Commands/SNTCommandStatus.mm

```diff
@@ -189,6 +190,30 @@ - (void)runWithArguments:(NSArray *)arguments {
           case SNTPushNotificationStatusDisabled: pushNotifications = @"Disabled"; break;
           case SNTPushNotificationStatusDisconnected: pushNotifications = @"Disconnected"; break;
           case SNTPushNotificationStatusConnected: pushNotifications = @"Connected"; break;
+          case SNTPushNotificationStatusConnectedNATS: {
+            pushNotifications = @"Connected (NPS Push Service)";
+            // Fetch the push server address for NATS
+            dispatch_semaphore_t serverSema = dispatch_semaphore_create(0);
+            [rop pushNotificationServerAddress:^(NSString *serverAddress) {
+              if (serverAddress) {
+                // Remove tls:// prefix and port number for display
+                NSString *cleanedAddress = serverAddress;
+                if ([cleanedAddress hasPrefix:@"tls://"]) {
+                  cleanedAddress = [cleanedAddress substringFromIndex:6];
+                }
+                // Remove port number (e.g., :443)
+                NSRange portRange = [cleanedAddress rangeOfString:@":"];
+                if (portRange.location != NSNotFound) {
+                  cleanedAddress = [cleanedAddress substringToIndex:portRange.location];
+                }
+                pushServerAddress = cleanedAddress;
+              }
```

Comment:
If the address looks like `scheme://my.server.com:port`, you can extract the `my.server.com` part via:
```suggestion
              if (serverAddress) {
                pushServerAddress = [[NSURL URLWithString:serverAddress] host];
              }
```